networking/synthetic/stac-on-k8s/stac-consumer.yml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: stac-consumer
+  annotations:
+    sfc-nic-ip: 70.70.70.2/24
```

Comment:
I am wondering if we want something to actually generate these yaml files...ansible to generate the yaml itself.